services/oada_upload/package.json

```diff
@@ -1,24 +1,30 @@
 {
   "name": "@isoblue/oada_upload",
+  "type": "module",
   "version": "0.0.1",
   "description": "Upload Avena time-series datapoints to an OADA store",
-  "main": "src/oada_upload.ts",
+  "exports": "./build/oada_upload.js",
   "repository": "https://github.com/oats-group/isoblue-avena",
   "author": "Aaron Neustedter",
   "contributors": [
     "Andrew Balmos <abalmos@purdue.edu>"
   ],
   "license": "MIT",
   "scripts": {
-    "start": "ts-node src/oada_upload.ts"
+    "oldstart": "ts-node --files src/oada_upload.ts",
```

Comment:
oldstart?

Reply:
that's the command that was used to start the service when we started working on it, but that command quit working so it got saved there.  It can be safely deleted now, good catch.